src/views/Login.vue

```diff
@@ -1,57 +1,101 @@
 <template>
```

Comment:
Use pug

Reply:
Yeah, as I mentioned in another comment, I will migrate all templates in one PR.